railties/lib/rails/api/task.rb

```diff
@@ -9,14 +10,21 @@ class Task < RDoc::Task
             README.rdoc
             lib/active_support/**/*.rb
           ),
-          exclude: "lib/active_support/vendor/*"
+          exclude: [
+            'lib/active_support/i18n_railtie.rb',
```

Comment:
Yeah, that is true. But in the case of connection adapters I think there are a lot of public API methods there.